repositories.md

```diff
@@ -39,37 +39,37 @@ Listing the public key of the Time Server in Director Targets metadata is necess
 If the Time Server is implemented, the primary is CONDITIONALLY REQUIRED to use the following procedure to verify the time. This procedure occurs after the vehicle version manifest is sent and will fulfill the ["Download and check current time"](https://uptane.github.io/papers/ieee-isto-6100.1.0.0.uptane-standard.html#check_time_primary) step of the Uptane Standard.
 
 1. Gather the tokens from each secondary ECU's version report.
-2. Send the list of tokens to the Time Server to fetch the current time. The Time Server responds as described in the [Time Server section](#time-server), providing a cryptographic attestation of the last known time.
-3. If the Time Server's response meets the criteria below, update the Primary ECU's clock and retain the Time Server's response for distribution to Secondary ECUs. Otherwise, the response will be discarded and procedure continues without an updated time.  The criteria for checking the Time Server's response are:
+2. Send the list of tokens to the Time Server to fetch the current time. The Time Server responds, as described in the [Time Server section](#time-server), by providing a cryptographic attestation of the last known time.
+3. If the Time Server's response meets the criteria below, update the Primary ECU's clock and retain the Time Server's response for distribution to Secondary ECUs. If it fails to meet this criteria, discard the response and continue the procedure without an updated time.  The criteria for checking the Time Server's response are:
   - The signature over the Time Server's response is valid.
-  - All the tokens provided to the Time Server have been included in the response.
+  - All the tokens provided to the Time Server are included in the response.
   - The time in the Time Server's response is later than the last time verified in this manner.
 
 #### ECU Version Report
 
-The ECU version report from each Secondary will contain a token to be sent to the Time Server. This token SHOULD be unique for each update cycle to prevent a replay. Since we expect updates to be relatively infrequent (e.g., due to limited number of write cycles), and that there will be a large number of possible tokens, it should be possible to produce a unique token for every update.
+The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. This stipulation should be possible to achieve as we expect updates to be relatively infrequent (e.g., due to limited number of write cycles), and that there will be a large number of possible tokens.
```

Comment:
Better, but it needs a little additional tweaks. How about this?
```suggestion
The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible"
```

Reply:
Yep, looks good, other than the stray quote mark instead of a period at the end!

Reply:
```suggestion
The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible.

```